packages/ckeditor5-bookmark/tests/bookmarkediting.js

```diff
@@ -24,10 +30,13 @@ describe( 'BookmarkEditing', () => {
 		editor = await ClassicTestEditor
 			.create( element, {
 				language: 'en',
-				plugins: [ BookmarkEditing, Enter, Paragraph, Undo, Link ]
+				plugins: [ BookmarkEditing, Enter, Image, Heading, Paragraph, Undo, Link ]
 			} );
 
 		model = editor.model;
+		view = editor.editing.view;
+		viewDocument = new ViewDocument( new StylesProcessor() );
+		converter = new DomConverter( viewDocument );
```

Comment:
Why not use those already created by the editor?

Reply:
Fixed in 2bdf291.